### DISCUSSION TITLE
Add Spirit of Competition pet to the CC rewards catalogue.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#4938] [Suggestions]
Currently the [Essence of Competition](https://wowgaming.altervista.org/aowow/?spell=48408) (and [Competitor's Tabard](https://wowgaming.altervista.org/aowow/?item=36941)) is on offer on the CC rewards catalogue. The pet was an Asia-only reward as a part of the 2008 summer Beijing Olympics promotion, with a North American equivalent of the [Spirit of Competition](https://wowgaming.altervista.org/aowow/?spell=48406). Unless somehow someway this event is re-implemented/re-contextualized on CC (which I'm not suggesting 😛) then I feel that both of these otherwise unobtainable companions should be available from the rewards catalogue.

**Comment by 55Honey** [comment]
Hey Bench, thanks for the suggestion.

We will keep looking into expanding the catalogue. However, for now this item in particular isn't scheduled yet for a certain occasion.
It might be in the catalogue eventually or used for an event maybe. I'm sure you'll notice as soon as it's out.
